### PR TITLE
feature/new-data

### LINKED ISSUE #17: new api
apple just dropped a new api https://developer.apple.com/tutorials/data/documentation/technologies.json probably worth using that instead of db.json

### PR DESCRIPTION
**Pull request by MichaelNeas**
Adds db2 to cardstore for updated data, adds js script to pull relevant information
close #17

### CODE REVIEW

**Comment by MichaelNeas** [comment]
@emyarod  added to review reducer.js

File: reducer.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+https.get(url, res => {
+	let body = "";
+	res.on("data", data => {
+		body += data;
+	});
+	res.on("end", () => {
+		body = JSON.parse(body);
+		let content = body.sections[1].groups
+		let references = body.references
+		let output = {}
```
Comment (emyarod) [inline reducer.js:14]:
```suggestion
		const content = body.sections[1].groups;
		const { references } = body;
		const output = {};
```

File: reducer.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+				output[content[i].name].push({
+					question: title,
+					answer: abstract,
+					link: refURL,
+					sdks: tags
+				})
+			}
+			
+		}
```
Comment (emyarod) [inline reducer.js:40]:
```suggestion
		content.forEach(group => {
			const { technologies, name } = group;
			
			output[name] = technologies.map(technology => {
				const { title: question, destination, tags: sdks } = technology;
				const { identifier: reference } = destination;
				let answer = '';
                                let link = '';
                                
				if (references[reference]) {
					answer = references[reference].abstract[0].text;
					link = `https://developer.apple.com${references[reference].url}`;
				}

				return {
					question,
					answer,
					link,
					sdks,
				};
			});
		});
```

File: Framewerk/Card.swift [hunk 3]
```diff
@@ -15,12 +15,26 @@ struct FramewerkCardData: Codable {
     let developerTools: [Card]
     let graphicsAndGames: [Card]
     let media: [Card]
+    let releaseNotes: [Card]
     let system: [Card]
     let web: [Card]
+    //let design: [Card]
```
Comment (emyarod) [inline Framewerk/Card.swift:21]: is this comment intentional?

File: Framewerk/Card.swift [hunk 4]
```diff
@@ -15,12 +15,26 @@ struct FramewerkCardData: Codable {
...
+    enum CodingKeys: String, CodingKey {
+        case appFrameworks = "App Frameworks"
+        case appServices = "App Services"
+        case developerTools = "Developer Tools"
+        case graphicsAndGames = "Graphics and Games"
+        case media = "Media"
+        case releaseNotes = "Release Notes"
+        case system = "System"
+        case web = "Web"
+        //case design = "Design"
```
Comment (emyarod) [inline Framewerk/Card.swift:36]: is this comment intentional?

**Review by emyarod — COMMENTED** [review]
destructured and renamed some variables in `reducer.js`, and converted the `for` loops to `forEach` and `map` methods, hopefully the result is nicer to read

**Pull request merged**